container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/implementations/vector/vector_config.go

```diff
@@ -23,9 +23,12 @@ func newVectorConfig(
 	listeningPortNumber uint16,
 	httpPortNumber uint16,
 	sinks logs_aggregator.Sinks,
+	shouldTurnOffPersistentVolumeLogsCollection bool,
 ) *VectorConfig {
-	reconciledSinks := map[string]map[string]interface{}{
-		logs_aggregator.DefaultSinkId: {
+	reconciledSinks := map[string]map[string]interface{}{}
+
+	if !shouldTurnOffPersistentVolumeLogsCollection {
```

Comment:
this is where the positives vs negatives thing makes it easier to reason about.

e.g. I parse this as, "I'm _not_ turning _off_ persistent volume logs collection"

compare with `if shouldEnablePersistentVolumeLogsCollection`